src/python/pants/backend/codegen/jaxb/jaxb_gen.py

```diff
@@ -12,12 +12,41 @@
 from pants.backend.jvm.targets.java_library import JavaLibrary
 from pants.backend.jvm.tasks.nailgun_task import NailgunTask
 from pants.base.exceptions import TaskError
+from pants.base.workunit import WorkUnitLabel
+from pants.java.jar.jar_dependency import JarDependency
 from pants.task.simple_codegen_task import SimpleCodegenTask
 
 
 class JaxbGen(SimpleCodegenTask, NailgunTask):
   """Generates java source files from jaxb schema (.xsd)."""
 
+  _XJC_MAIN = 'com.sun.tools.xjc.Driver'
+
+  @classmethod
+  def register_options(cls, register):
+    super(JaxbGen, cls).register_options(register)
+    cls.register_jvm_tool(register,
+                          'xjc',
+                          classpath=[
+                            JarDependency(org='com.sun.xml.bind',
+                                          name='jaxb-core',
+                                          rev='2.3.0'),
```

Comment:
Are the `2.3.0` values all associated? Would be good to have a shared local variable for them.